storage/src/mmr/grafting.rs

```diff
@@ -753,118 +755,119 @@ mod tests {
                         .unwrap();
                     assert!(proof.verify_element_inclusion(
                         &mut verifier,
                         &b4,
-                        pos,
+                        loc,
                         &grafted_storage_root
                     ));
                 }
 
                 // Confirm element inclusion proof verification fails for various manipulations of the input.
                 {
                     // Valid proof of the last element.
-                    let pos = 4;
-                    let proof = verification::range_proof(&grafted_mmr, pos, pos)
+                    let loc = 3;
+                    let proof = verification::range_proof(&grafted_mmr, loc..loc + 1)
                         .await
                         .unwrap();
                     let mut verifier = Verifier::<Sha256>::new(GRAFTING_HEIGHT, 1, vec![&p2]);
                     assert!(proof.verify_element_inclusion(
                         &mut verifier,
                         &b4,
-                        pos,
+                        loc,
                         &grafted_storage_root
                     ));
 
                     // Proof should fail if we try to verify the wrong leaf element.
                     assert!(!proof.verify_element_inclusion(
                         &mut verifier,
                         &b3,
-                        pos,
+                        loc,
                         &grafted_storage_root
                     ));
 
                     // Proof should fail if we use the wrong root.
-                    assert!(!proof.verify_element_inclusion(&mut verifier, &b4, pos, &peak_root));
+                    assert!(!proof.verify_element_inclusion(&mut verifier, &b4, loc, &peak_root));
 
                     // Proof should fail if we use the wrong position
                     assert!(!proof.verify_element_inclusion(
                         &mut verifier,
                         &b4,
-                        3,
+                        loc + 1,
                         &grafted_storage_root
                     ));
 
                     // Proof should fail if we inject the wrong peak element into the verifier.
-                    let mut verifier = Verifier::<Sha256>::new(GRAFTING_HEIGHT, 1, vec![&p1]);
+                    let mut verifier = Verifier::<Sha256>::new(GRAFTING_HEIGHT, 0, vec![&p1]);
                     assert!(!proof.verify_element_inclusion(
                         &mut verifier,
                         &b4,
-                        pos,
+                        loc,
                         &grafted_storage_root
                     ));
 
                     // Proof should fail if we give the verifier the wrong peak tree leaf number.
-                    let mut verifier = Verifier::<Sha256>::new(GRAFTING_HEIGHT, 2, vec![&p1]);
+                    let mut verifier = Verifier::<Sha256>::new(GRAFTING_HEIGHT, 0, vec![&p1]);
```

Comment:
This assertion is the same as the one above on line 850

Reply:
thx, fixed.